index.html

```diff
@@ -0,0 +1,240 @@
+      <section class="wrap pb-5">
+        <header class="menu-wrap">
+          <nav>
+            <input
+              type="checkbox"
+              class="toggler"
+              id="toggle"
+              onclick="toggle()"
+            />
+            <div class="hamburger"><div></div></div>
+            <div class="menu">
+              <div class="overlay">
+                <div>
+                  <ul class="menubar">
+                    <li>
+                      <a href="#" class="brand">
+                        <img src="./img/logo.png" alt="logo" />
+                      </a>
+                    </li>
+                    <li>
+                      <ul class="nav-wrap">
+                        <li>
+                          <a href="about.html" class="nav-links about-link">
+                            About
+                          </a>
+                        </li>
+                        <li><a href="#" class="nav-links">Program</a></li>
+                        <li>
+                          <a href="#works" class="nav-links">Buy Tickets</a>
+                        </li>
+                        <li><a href="#works" class="nav-links">Sponsor</a></li>
+                        <li><a href="#works" class="nav-links">News</a></li>
+                        <li class="cc">
+                          <a href="index.html" class="textcc">
+                            Fundraising Concert
+                          </a>
+                        </li>
+                      </ul>
+                    </li>
+                  </ul>
+                </div>
+              </div>
+            </div>
+          </nav>
+        </header>
+        <div class="intro">
+          <p id="introText">"Hello! Sharing World"</p>
+          <h1 class="heading mb-4">
+            Fund Raising Concert
+            <br />
+            for Displaced People
+            <br />
+            in Myanmar 2023
+          </h1>
+        </div>
+        <div class="mb-sm-5 detail-wrap">
+          <p class="detail p-2">
+            A fund raising concert for all the Displaced people in Myanmar after
+            the coup, creating a positive change with people from all over the
+            country and around the world is taking place in April, in Myanmar
+          </p>
+          <h2 class="day">2023.04.07(FRI) ~ 10(SUN)</h2>
+          <p class="place">@National Kandawgyi Gardens</p>
+        </div>
+      </section>
+      <section class="mainProgram pb-lg-4">
+        <p class="title pt-4 pb-4 text-center">Main Program</p>
+        <div class="program-wrap">
+          <div class="program">
+            <div class="transbox">
+              <i class="fa-solid fa-pen-fancy fa-xl icon"></i>
+              <p class="program-title">Autographs</p>
+              <p class="programDetail">
+                Meet the famous artists, get their autographs and taking photos
+                with your idols
+              </p>
+            </div>
```

Comment:
- [ ] Each of these cards should be wrapped in an `article` tag as these cards are articles.